include/FockSpace/BaseFockSpace.hpp

```diff
@@ -29,10 +29,12 @@ namespace GQCP {
 
 
 /**
- *  A base class for the Fock space
- *  Interfacing requires the Fock space to generate an ONV from a given address
- *  transform a given ONV into the next ONV (in the full or selected space)
- *  and retrieve the address of a given ONV in the space
+ *  A base class for the representation of a Fock space.
+ *
+ *  Derived classes should be able to
```

Comment:
No longer the case. This should be removed.  The Derived classes no longer interface this way to the base class.